fab/fabfile.py

```diff
@@ -367,6 +367,14 @@ def install_packages():
 
 
 @task
+@roles(ROLES_TOUCHFORMS)
+def install_npm_packages():
+    """Install required NPM packages for server"""
+    with cd(os.path.join(env.code_root, 'submodules/touchforms-src/touchforms')):
+        sudo("npm install")
```

Comment:
I guess this is no different from running pip, so probably not a big deal since that can have the same connectivity issues. I also see now that this only runs as part of initial server setup. That's nice that it doesn't need to be run on every deploy.